Resources/Keywords/Navigation.resource

```diff
@@ -0,0 +1,27 @@
+*** Settings ***
+Library    String
+Resource    ../OR/Login_OR.resource
+Resource    ../Keywords/Login.resource
+
+*** Keywords ***
+User Navigates to ${subMenuOption} under ${sideNavBarMenu}
+    User Expands ${sideNavBarMenu}
+    User Selects ${subMenuOption} on ${sideNavBarMenu}
+
+User Expands ${sideNavBarMenu}
+    Log To Console    ${sideNavBarMenu}
+    ${eleNavMenu}=  Replace String  ${eleNavMenu}   <<menu>>    ${sideNavBarMenu}
+    Element Should Be Visible    ${eleNavMenu}
+    Click Element    ${eleNavMenu}
+
+User Selects ${subMenuOption} on ${sideNavBarMenu}
+    ${eleNavChildMenu}=     Replace String  ${eleNavChildMenu}  <<menu>>    ${sideNavBarMenu}
+    ${eleNavChildMenu}=     Replace String  ${eleNavChildMenu}  <<submenu>>    ${subMenuOption}
+    Element Should Be Visible    ${eleNavChildMenu}
+    Click Element    ${eleNavChildMenu}
+    ${lblChildPageHeader}=     Replace String    ${lblChildPageHeader}  <<submenu>>    ${subMenuOption}
```

Comment:
Why is this element called lblChildPageeader? Don't understand what it represents. Seems a bit too long and confusing to me.

Reply:
@ikanias it represents label on the child page header, it is on the openshift left navigation menu. For example, if the user selects Operator --> Installed Operator then it is the label on the child page header.